src/ThreeEditor/js/EditorOrbitControls.js

```diff
@@ -0,0 +1,38 @@
+import * as THREE from 'three'
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
+
+export class EditorOrbitControls extends OrbitControls {
+    box = new THREE.Box3();
+    sphere = new THREE.Sphere();
+    delta = new THREE.Vector3();
+
+    focus(target) {
```

Comment:
Add some comments explaining the logic here.
`distance` is between what objects ?
What is the reason behind the `if` switch ?
What is the magic `0.1` distance ?

Reply:
Code in this file was copied from `EditorControls.js`, because in OrbitControls `focus` function was missing.
(OrbitControls supports ortho camera).
(EditorControls was first controls used in the editor)

Reply:
Great, so please add this information (that the code was copied and why!) as the comment.
Also mention exactly from where was it copied (i.e. mentioning version of threejseditor and location of original source file)

Reply:
Looks good